Update README.md (#1429)
openai/evals: README.md

# OpenAI Evals

> Evals provide a framework for evaluating large language models (LLMs) or systems built using LLMs. We offer an existing registry of evals to test different dimensions of new models and the ability to write your own custom evals for use cases you care about. You can use your data to build private evals which represent the common patterns you use LLMs for without exposing any of that data publically.

Author: ```suggestion
Evals provide a framework for evaluating large language models (LLMs) or systems built using LLMs. We offer an existing registry of evals to test different dimensions of OpenAI models and the ability to write your own custom evals for use cases you care about. You can also use your data to build private evals which represent the common LLMs patterns in your workflow without exposing any of that data publicly.
```

> If you are building with LLMs, creating high quality evals is one of the most impactful things you can do. With evals, it can be very difficult and time intensive to understand how different model versions might effect you use case. In the words of [OpenAI's President Greg Brockman](https://twitter.com/gdb/status/1733553161884127435):

Author: ```suggestion
If you are building with LLMs, creating high quality evals is one of the most impactful things you can do. With evals, it can be very difficult and time intensive to understand how different model versions might effect your use case. In the words of [OpenAI's President Greg Brockman](https://twitter.com/gdb/status/1733553161884127435):
```

> If you are building with LLMs, creating high quality evals is one of the most impactful things you can do. With evals, it can be very difficult and time intensive to understand how different model versions might effect your use case. In the words of [OpenAI's President Greg Brockman](https://twitter.com/gdb/status/1733553161884127435):

Author: ```suggestion
If you are building with LLMs, creating high quality evals is one of the most impactful things you can do. Without evals, it can be very difficult and time intensive to understand how different model versions might effect your use case. In the words of [OpenAI's President Greg Brockman](https://twitter.com/gdb/status/1733553161884127435):
```

If you want to manually run all pre-commit hooks on a repository, run `pre-commit run --all-files`. To run individual hooks use `pre-commit run <hook_id>`.

## Running evals

If you don't want to contribute new evals, but simply want to run them locally, you can install the evals package via pip:

```sh
pip install evals
```

> You can find the full instructions to run existing evals in: [run-evals.md](docs/run-evals.md) and our existing eval templates: [eval-templates.md](docs/eval-templates.md). For more advanced use cases like prompt chains or tool-using agents, you can use our: [Completion Function Protocol](docs/completion-fns.md).

Author: ```suggestion
You can find the full instructions to run existing evals in: [run-evals.md](docs/run-evals.md) and our existing eval templates in: [eval-templates.md](docs/eval-templates.md). For more advanced use cases like prompt chains or tool-using agents, you can use our: [Completion Function Protocol](docs/completion-fns.md).
```
